.github/workflows/quarkus-snapshot.yaml

```diff
@@ -18,30 +18,51 @@ env:
 
   ECOSYSTEM_CI_REPO_PATH: quarkiverse-artemis
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
 jobs:
   build:
     name: "Build against latest Quarkus snapshot"
     runs-on: ubuntu-latest
     # Allow <ADMIN> to manually launch the ecosystem CI in addition to the bots
-    if: github.actor == 'quarkusbot' || github.actor == 'quarkiversebot' || github.actor == '<ADMIN>'
 
     steps:
+      - name: Check user for team affiliation
+        uses: tspascoal/get-user-teams-membership@v2
+        id: teamAffiliation
+        with:
+          GITHUB_TOKEN: ${{ secrets.ECOSYSTEM_CI_TOKEN }}
+          username: ${{ github.actor }}
+          team: quarkiverse-artemis
+
+      - name: Stop workflow if user is no member
+        if: ${{ steps.teamAffiliation.outputs.isTeamMember == false }}
+        run: |
+          echo "You have no rights to trigger this job."
+          exit 1
+
```

Comment:
I think we can leave that out until we figure out what's wrong with the token.

Reply:
@gastaldi removed. Care to take another look?